android/build.gradle

```diff
@@ -81,12 +81,12 @@ android {
 
   sourceSets {
     main {
-      if (isNewArchitectureEnabled()) {
+      //if (isNewArchitectureEnabled()) {
           java.srcDirs += [
             "generated/java",
             "generated/jni"
           ]
-      }
+      //}
```

Comment:
When we support old architecture and new architecture in the same package, we will need to revisit this.